Apps/WebClient/src/NewClientApp/src/components/modal/AppTourComponent.vue

```diff
@@ -0,0 +1,222 @@
+                            <v-col>
+                                <h3 class="text-h6 font-weight-bold">
+                                    {{ currentSlide.title }}
+                                </h3>
+                                <p
+                                    v-if="currentSlide.description"
+                                    class="text-body-1"
+                                >
+                                    {{ currentSlide.description }}
+                                </p>
+                            </v-col>
+                        </v-row>
+                        <v-row v-if="isFirstSlide" class="mt-4">
+                            <v-col cols="3">
+                                <HgButtonComponent
+                                    variant="link"
+                                    data-testid="app-tour-skip"
+                                    @click="hideModal"
+                                    text="Skip"
+                                />
+                            </v-col>
+                            <v-col cols="6" class="d-flex justify-center">
+                                <HgButtonComponent
+                                    variant="primary"
+                                    data-testid="app-tour-start"
+                                    @click="next()"
+                                    text="Start Tour"
+                                />
+                            </v-col>
+                        </v-row>
+                        <v-row
+                            v-else-if="!isFirstSlide && !isFinalSlide"
+                            class="mt-3"
+                        >
+                            <v-col>
+                                <HgButtonComponent
+                                    variant="link"
+                                    data-testid="app-tour-skip"
+                                    @click="hideModal"
+                                    text="Skip"
+                                />
+                            </v-col>
+                            <v-col class="d-flex justify-end">
+                                <HgButtonComponent
+                                    data-testid="app-tour-back"
+                                    variant="secondary"
+                                    @click="previous()"
+                                    text="Back"
+                                />
+                                <HgButtonComponent
+                                    variant="primary"
+                                    class="ml-4"
+                                    data-testid="app-tour-next"
+                                    @click="next()"
+                                    text="Next"
+                                />
+                            </v-col>
+                        </v-row>
+                        <v-row v-else class="mt-6">
+                            <v-col class="d-flex justify-center">
```

Comment:
Row & col wrapper here don't seem to be needed - could be replaced with `<div>`.